dev/ci/user-overlays/13448-ybertot-fix_simpl_never_wrt_delta.sh

```diff
@@ -0,0 +1,3 @@
+overlay hott https://github.com/ybertot/HoTT respect_simpl_never_flags 13448
```

Comment:
It seems this overlay was missed, I made https://github.com/HoTT/Coq-HoTT/pull/1713